src/main/kotlin/FinalProject/Main.kt

```diff
@@ -1,5 +1,93 @@
 package FinalProject
 
+import com.univocity.parsers.csv.CsvParser
+import com.univocity.parsers.csv.CsvParserSettings
+import com.univocity.parsers.common.record.Record
+import com.univocity.parsers.csv.CsvWriter
+import com.univocity.parsers.csv.CsvWriterSettings
+
+/* the scenario function creates a dozen cars, puts them into a road network,
+ * simulates the traffic and prints the jammed / non-jammed for every car.
+ */
+fun scenario(Cars: MutableList<Car>) {
+	/*val car1 = Car(true)
+	val car2 = Car(false)
+	val car3 = Car(true)
+	val car4 = Car(true)
+	val car5 = Car(false)
+	val car6 = Car(true)
+	val car7 = Car(true)
+	val car8 = Car(false)
+	val car9 = Car(true)
+	val car10 = Car(true)
+	val car11 = Car(false)
+	val car12 = Car(true) */
+	
+	//val Cars = listOf(car1, car2, car3, car4, car5, car6, car7, car8, car8, car9, car10, car11, car12)
+	val rn = RoadNetwork(7, Cars)
+	rn.simulationStep()
+	rn.printCarsJammed()
+}
+
 fun main(args: Array<String>) {
-    println("Hello, World!")
+	val Cars : MutableList<Car> = parseCSV()
+
+	scenario(Cars)

```

Comment:
@nicolaiparlog 
The scenario function has a parameter which takes the car list, that means the scenario function itself is independent of CSV. The main function handles the CSV reading/writing